### PR TITLE
RavenDB-21504 Forcing update of expired license is successful but doe…

### PR DESCRIPTION
**Pull request by efratshenhar**
…sn't work

### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-21504

### Additional description

Return information when the license update was unsuccessful.

### Type of change

- Bug fix

### How risky is the change?

- Low 

### Backward compatibility

- Non breaking change

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing by Contributor

- It has been verified by manual testing

### Testing by RavenDB QA team

- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- It requires further work in the Studio. Please mark the issue on YouTrack using `Studio Required` tag.

### CODE REVIEW

File: src/Raven.Server/Web/Studio/LicenseHandler.cs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Net;
+using System.Security.Cryptography;
```
Comment (grisha-kotler) [inline src/Raven.Server/Web/Studio/LicenseHandler.cs]: Is it needed?

File: src/Raven.Server/Web/Studio/LicenseHandler.cs [hunk 2]
```diff
@@ -98,9 +99,16 @@ public async Task ForceUpdate()
...
-
-            NoContentStatus();
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                var updated = await ServerStore.LicenseManager.LeaseLicense(GetRaftRequestIdFromQuery(), throwOnError: true);
+                context.Write(writer, new DynamicJsonValue
+                {
+                    ["Updated"] = updated,
+                    ["Message"] = updated ? "License was updated Successfully" : "Unable to update license"
```
Comment (grisha-kotler) [inline src/Raven.Server/Web/Studio/LicenseHandler.cs]:
```suggestion
                    ["Message"] = updated ? "License was updated Successfully" : "The license wasn't modified"
```

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Web/Studio/LicenseHandler.cs (hunk 1 above)
Comment (efratshenhar) [reply src/Raven.Server/Web/Studio/LicenseHandler.cs]: no. removed

File: src/Raven.Server/Web/Studio/LicenseHandler.cs (hunk 2 above)
Comment (efratshenhar) [reply src/Raven.Server/Web/Studio/LicenseHandler.cs]: done

File: src/Raven.Server/Web/Studio/LicenseHandler.cs [hunk 3]
```diff
@@ -98,9 +98,16 @@ public async Task ForceUpdate()
...
             }
 
-            await ServerStore.LicenseManager.LeaseLicense(GetRaftRequestIdFromQuery(), throwOnError: true);
-
-            NoContentStatus();
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                var updated = await ServerStore.LicenseManager.LeaseLicense(GetRaftRequestIdFromQuery(), throwOnError: true);
+                context.Write(writer, new DynamicJsonValue
```
Comment (grisha-kotler) [inline src/Raven.Server/Web/Studio/LicenseHandler.cs]: Let's add a class with an enum with 2 states:
`Updated` and `NotModified` and return that.

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Web/Studio/LicenseHandler.cs (hunk 3 above)
Comment (efratshenhar) [reply src/Raven.Server/Web/Studio/LicenseHandler.cs]: done

**Review by grisha-kotler — APPROVED** [review]

**Pull request merged**